google-cloud-spanner/test/google/cloud/spanner/client/transaction_retry_test.rb

```diff
@@ -24,8 +24,8 @@
   let(:transaction_id) { "tx789" }
   let(:transaction_grpc) { Google::Cloud::Spanner::V1::Transaction.new id: transaction_id }
   let(:transaction) { Google::Cloud::Spanner::Transaction.from_grpc transaction_grpc, session }
-  let(:tx_selector) { Google::Cloud::Spanner::V1::TransactionSelector.new id: transaction_id }
-  let(:tx_selector_begin) do
+  let(:tx_id_selector) { Google::Cloud::Spanner::V1::TransactionSelector.new id: transaction_id }
+  let(:tx_begin_selector) do
```

Comment:
Why is the variable name changed? What are you trying to depict here? 
tx_selector_begin indicates tx_selector begin block 
tx_begin_selector indicates that you are selector is for tx_begin.
What's the intent here? I feel the previous variable should remain.

Reply:
The intent was to distinguish between `id` and `begin` transaction selectors in the tests. This wasn't an issue until ILB was implemented last year. Before that, we exclusively used id based selectors.

I rename the fixtures from `tx_id_selector` to `tx_selector_id` (same for `tx_begin_selector`)